redirects.js

```diff
@@ -52,6 +52,67 @@ const setup = (app) => {
   conflates a language part ("zh") with a region ("europe") so we want to keep zh out of the manifest JSON */
   app.route("/ncov/zh").get((req, res) => res.redirect("/ncov/global/zh"));
 
+  // DOCS, HELP Redirects for docs migration:
+  const mainReadTheDocs = "https://docs.nextstrain.org/en/latest";
+  const augurReadTheDocs = "https://docs.nextstrain.org/projects/augur/en/migrate-docs";
```

Comment:
Leaving a placeholder "Todo" here as this should be /latest but this requires the Augur PR to be merged